apps/github-cascading-app/src/cascading/templates/pull-request-content.ejs

```diff
@@ -51,7 +51,7 @@ You can fix the issue locally with the following commands:
   and update this Pull Request with
 
   ```shell
-  git push origin <%= targetBranch %>
+  git push origin <%= originBranchName %>
```

Comment:
I had to run `git push origin HEAD:<% originBranchName %>` for it to work in local

Reply:
maybe 
`git push --set-upstream origin <%= originBranchName %>`